pkg/connector/startchat.go

```
@@ -0,0 +1,106 @@
+package connector
+
+import (
+	"context"
+	"fmt"
+
+	"go.mau.fi/util/ptr"
+	"maunium.net/go/mautrix/bridgev2"
+	"maunium.net/go/mautrix/bridgev2/database"
+	"maunium.net/go/mautrix/bridgev2/networkid"
+	"maunium.net/go/mautrix/event"
+
+	"go.mau.fi/mautrix-linkedin/pkg/linkedingo"
+)
+
+var (
+	_ bridgev2.GhostDMCreatingNetworkAPI = (*LinkedInClient)(nil)
+	_ bridgev2.GroupCreatingNetworkAPI   = (*LinkedInClient)(nil)
+)
+
+func (l *LinkedInClient) ResolveIdentifier(ctx context.Context, identifier string, createChat bool) (*bridgev2.ResolveIdentifierResponse, error) {
+	id := networkid.UserID(identifier)
+	ghost, _ := l.main.Bridge.GetGhostByID(ctx, id)
```

Errors should be propagated rather than ignored